lib/spack/spack/test/cmd/dev_build.py

```diff
@@ -96,7 +93,7 @@ def test_dev_build_until_last_phase(tmpdir, install_mockery):
     assert os.path.exists(str(tmpdir))
 
 
-def test_dev_build_before_until(tmpdir, install_mockery, capsys):
+def test_dev_build_before_until(tmpdir, install_mockery):
```

Comment:
I assume this wasn't essential to getting the tests to run on Windows, but was removed because it isn't necessary in the test (if so, then that works - I wanted to make sure I understood).

Reply:
The capsys parameter caused an error on Windows and was not used in the function. Its removal allowed it to work on Windows.

Reply:
Thanks! I didn't realize (or forgot) that capsys is unavailable on Windows. Anyhow not a problem since - as you say - it is not actually used in this test.